util/chunk/column.go

```diff
@@ -258,6 +258,13 @@ func (c *Column) resize(n, typeSize int) {
 	} else {
 		c.nullBitmap = make([]byte, sizeNulls)
 	}
+	var nullVal byte = 0
+	if !isNull {
+		nullVal = 0xFF
+	}
+	for i := range c.nullBitmap {
+		c.nullBitmap[i] = nullVal
+	}
```

Comment:
unnecessary initialization if `isNull` is true.

Reply:
updated.